plugins/zengame/includes/API/class-rest-handler.php

```
@@ -243,95 +252,172 @@ private static function get_user_rank_data(int $user_id): array
+
+        return \array_values(\array_filter($types, static fn($type) => \is_array($type) && !empty($type['slug'])));
+    }
+
+    private static function get_next_rank_post($current_rank, string $rank_slug): ?\WP_Post
+    {
+        if ($rank_slug == '') {
+            return null;
+        }
+
+        $current_menu_order = ($current_rank && isset($current_rank->menu_order)) ? (int) $current_rank->menu_order : -1;
+        $rank_posts = \get_posts([
```

PHPMD (2.15.0)

[error] The variable $rank_posts is not named in camelCase. (undefined)

(CamelCaseVariableName)